SOOP/SOOP_CO2/dest_path.py

```diff
@@ -68,46 +136,46 @@ def soop_co2_dest_path(nc_file):
             return None
```

Comment:
line 135 , the nc_file variable does not exist
print >>sys.stderr, "Hierarchy not created for '%s'" % nc_file, should be file I guess